### ISSUE TITLE
Inconsistent include directory structure between "build" and "install" (CMake)

### ISSUE THREAD

**Issue by cstlee** [PlatformLab/PerfUtils#14]
Applications of PerfUtils using CMake need to include headers differently depending on whether the application chooses to compile against the "build" copy of PerfUtils or the "install" copy of PerfUtils.  At the moment if an application wants to include `Cycles.h` using the "build" copy, the application would `#include "Cycles.h"`; if using the "install" copy, the application would `#include "PerfUtils/Cycles.h"`.  This means that applications source code can't be agnostic to how a user might want to make PerfUtils available in their build.

In short, it would be beneficial if the include statements for PerfUtils headers were the same regardless of a users installation preferences.
